### PR TITLE
A series of trivial changes to the main README

### PR DESCRIPTION
**Pull request by AaronCritchley**
A few questions:
- I noticed some inconsistencies in spacing after a sentence ends, is the preference to use one or two spaces? Let me know and I'll change them. 
- Code blocks for the most part are not using the laguage definition, one at the bottom is using the Bash 'lint', I think the majority of them should. Again, if I can get confirmation on The Guardians preference I will be happy to implement.

:smile:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 We're hiring!
 --------
-Ever thought about joining us?<br/>
```
Comment (jbreckmckye) [inline README.md]: The explicit <br/> is needed to force the line break. If we don't want a break, put it all on one line.

Comment (AaronCritchley) [inline README.md]: Are you sure? The preview still shows it as being broken line by line and I've never come across .md *needing* the explicit tag, if it's being used somewhere else and that's why it needs it, let me know and I'll add it in :+1:

Comment (jbreckmckye) [inline README.md]: > Are you sure?

Yep. Take a look in the browser.

**Comment by jbreckmckye** [comment]
1. Browsers don't typically display redundant whitespace in CDATA. As they are displayed as one space, I would just use one likewise.
2. I'm quite sure if a cabal of top developers at El Grauniad have ever laid out a policy on such things, but I don't think it's necessary. It's mostly boilerplate and I don't really think the syntax highlighting is really going to help anyone just copypasting instructions into their command line. Let's keep the docs easy to edit.

Aside from the comment, this seems fine. :+1:

**Comment by AaronCritchley** [comment]
Comment's been adressed, let me know if you have any other problems, I'll be happy to fix :+1:

**Comment by piuccio** [comment]
nice :+1:

**Comment by rich-nguyen** [comment]
thank you @AaronCritchley, much appreciated!

**Pull request merged**